src/extension.ts

```diff
@@ -276,7 +276,7 @@ export async function activate(context: vscode.ExtensionContext) {
 // This is only being used for errors from commands although there's plenty of other places where it
 // could be used. It needs to be apart of some sort of `core` module that is guaranteed to initialize
 // prior to every other Toolkit component. Logging and telemetry would fit well within this core module.
-async function handleError(error: unknown, topic: string, defaultMessage: string) {
+export async function handleError(error: unknown, topic: string, defaultMessage: string) {
```

Comment:
instead of exporting this should we keep the same pattern where extension.ts calls `registerErrorHandler` in the relevant module? that will keep all the related code in one place